Adds Symfony 5.0 compatibility.

kiettran opened the pull request.

niels-nijens commented:
Thanks for your contribution! Unfortunately, I expect this will need some more effort to get the bundle to work with both Symfony 3.4, 4.x and 5.x than only changing the Composer requirements.

kiettran commented:
Apparently there are some issues for adding compatibility to Symfony 5.0.
Due to some Deprecated Events in the HttpKernel (in 4.3), it will be hard to apply fully support for Symfony 5.0 without having two different project versions.

The following Events causes that problem.

**GetResponseEvent** (in [JsonRequestBodyValidationSubscriber](https://github.com/nijens/openapi-bundle/blob/master/src/EventListener/JsonRequestBodyValidationSubscriber.php#L74))
Deprecated in 4.3, use RequestEvent instead. Removed in 5.0.

**GetResponseForExceptionEvent** (in [JsonResponseExceptionSubscriber](https://github.com/nijens/openapi-bundle/blob/master/src/EventListener/JsonResponseExceptionSubscriber.php#L52))
Deprecated in 4.3, use ExceptionEvent instead. Removed in 5.0.

**FileLoaderLoadException** (in [SchemaLoader](https://github.com/nijens/openapi-bundle/blob/master/src/Json/SchemaLoader.php#L76))
Deprecated in 4.2, use LoaderLoadException instead. Removed in 5.0.

kiettran commented:
This pull request is outdated and does not add fully support for symfony 5.
